build/commands/lib/config.js

```diff
@@ -374,6 +374,7 @@ Config.prototype.buildArgs = function () {
     args.allow_runtime_configurable_key_storage = true
     if (this.use_goma && this.gomaServerHost) {
       args.use_system_xcode = false
+      args.enable_precompiled_headers = false
```

Comment:
Let's do this in another PR. This change will force a full rebuild if you want to switch between goma/local compilation on **all** platforms. Currently the full rebuild is triggered only on MacOS because of `use_system_xcode=false`.

TBH I don't think we have to disable PCHs, because goma compiles everything pretty good in the current setup  and we can enable/disable goma at any point pretty easily.

Reply:
This change is currently enable only on MacOS which is already does a full rebuild with/without goma, so it's fine then for now.